pyblish_maya/lib.py

```diff
@@ -313,3 +310,28 @@ def _show_no_gui():
 
     messagebox.setStandardButtons(messagebox.Ok)
     messagebox.exec_()
+
+
+class Dock(QtWidgets.QWidget):
+
+    def __init__(self, dock_count, parent=None):
+        super(Dock, self).__init__(parent)
+        QtWidgets.QVBoxLayout(self)
+        object_name = "pyblish_maya.dock" + str(dock_count)
+        self.setObjectName(object_name)
+
+
+def dock(window):
+
+    main_window = None
+    for obj in QtWidgets.qApp.topLevelWidgets():
+        if obj.objectName() == "MayaWindow":
+            main_window = obj
+
+    self._dock_count += 1
```

Comment:
I don't know :) But I would probably rework the docking to tear down any existing dock and create a new one.

Maybe we should get some more opinions on whether ```pyblish-lite``` should be shown once or multiple times?